packages/devtools/lib/src/ui/icons.dart

```diff
@@ -127,6 +129,10 @@ class FlutterIcons {
   // Icons matching IntelliJ core icons.
   static const Icon locate = UrlIcon('/icons/general/locate.png');
   static const Icon forceRefresh = UrlIcon('/icons/actions/forceRefresh.svg');
+  // TODO: Do we want these dynamic? :/
+  static Icon get refresh => isDarkTheme
```

Comment:
Add a TODO to create a ThemedIcon class that will handle this case like we handle the themed color case.